requirements.txt

```diff
@@ -5,6 +5,7 @@ glymur
 imagecodecs
 jupyterlab
 matplotlib
+nbsphinx
```

Comment:
do we need this outside dev environment?

Reply:
Resolved in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/157/commits/06b2d319969a284b19c4c631c72a11fd5c2dc632